pkg/apis/componentconfig/types.go

```diff
@@ -195,6 +195,9 @@ type KubeletConfiguration struct {
 	// podManifestPath is the path to the directory containing pod manifests to
 	// run, or the path to a single manifest file
 	PodManifestPath string
+	// checkpointPath is the path to the directory where the kubelet will checkpoint
+	// annotated pod manifests and configmaps before they are run
```

Comment:
Not implemented yet, right?
Should we consider adding this to the comment once we actually add this feature?

Reply:
Fixed.